cmd/antrea-controller/controller.go

```diff
@@ -112,8 +119,8 @@ func run(o *Options) error {
 	stopCh := signals.RegisterSignalHandlers()
 
 	informerFactory.Start(stopCh)
-	// Only start watching Security CRDs when ClusterNetworkPolicy is enabled.
-	if features.DefaultFeatureGate.Enabled(features.ClusterNetworkPolicy) {
+	// Only start watching CRDs when ClusterNetworkPolicy or Traceflow is enabled.
+	if features.DefaultFeatureGate.Enabled(features.ClusterNetworkPolicy) || features.DefaultFeatureGate.Enabled(features.Traceflow) {
```

Comment:
Added feature gate for traceflow informer.
```
	var traceflowInformer opsinformers.TraceflowInformer
	if features.DefaultFeatureGate.Enabled(features.Traceflow) {
		traceflowInformer = crdInformerFactory.Ops().V1alpha1().Traceflows()
	}
```

Reply:
I confirmed from informer code that it will only start watching a resource if a resource's `Informer()` is called. But CNP code calls `Informer()` unconditionally and rely on `crdInformerFactory.Start(stopCh)` to not start watching it. So if this is merged, enabling TF could cause confusing error logs to CNP.
But perhaps I can handle this after it's merged. Can you confirm if there are error logs about CNP watching if enabling TF?
In this PR, you can create `traceflowInformer` unconditionally given other feature gated Informer already does it and it's harmless.

Reply:
I found no error logs related to CNP if only enabling TF.

Reply:
I see, it's because the CRD and RBAC is applied unconditionally.
The side effect is ClusterNetworkPolicy will be enabled as long as TF feature is enabled. But this should be fixed in CNP code.